<launch>
    <arg name="video_stream_provider" default="0" />
    <arg name="camera_connector_chart" default="" />
    <arg name="python_node" default="false" />
    <arg name="realsense" default="true" />

    <include file="$(find interbotix_xsarm_control)/launch/xsarm_control.launch">
        <arg name="robot_model" value="$(env ROBONETV2_ARM)"/>
        <arg name="use_rviz" value="False"/>
    </include>


    <group if="$(arg realsense)">
        <include file="$(find robonetv2)/launch/realsense.launch" />
    </group>
    <group unless="$(arg realsense)">
        <include file="$(find multicam_server)/launch/cameras.launch">
    	  	<arg name="video_stream_provider" value="$(arg video_stream_provider)" />
    	  	<arg name="camera_connector_chart" value="$(arg camera_connector_chart)" />
        </include>
    </group>
</launch>
